source/account_structure/set_up_where_payments_go/index.html.md.erb

```diff
@@ -0,0 +1,58 @@
+---
+title: Set up which bank accounts your payments go to
+last_reviewed_on: 2021-02-10
+review_in: 6 months
+weight: 141
+---
+
+# Set up which bank accounts your payments go to
+
+To help you reconcile payments, you can send payments from your services to:
+
+- the same bank account
+- different bank accounts
+
+GOV.UK Pay services are free, and there's no limit to how many you can create. Your payment service provider (PSP) may charge you for each PSP account you have, unless you use GOV.UK Pay's PSP.
```

Comment:
The phrasing here sort of makes the two things (does your PSP charge you / does Stripe charge you) feel more dependent on each other? I'd be tempted to separate it out and just say - there's no charge for GOV.UK Pay's PSP.

Reply:
Thank you - updated in 160e014